styles/signals.mapcss

```diff
@@ -69,10 +69,18 @@ way["railway:lzb"=yes].tracks
 	z-index: 3;
 	color: red;
 }
+way["railway:atb"=yes].tracks
+way["railway:atb-eg"=yes].tracks
+way["railway:atb-ng"=yes].tracks
+way["railway:atb-vv"=yes].tracks
+{
+	z-index: 4;
+	color: #FF8C00;
+}
 way["construction:railway:etcs"]["construction:railway:etcs"!=no]["construction:railway:etcs"!=0].tracks,
 way.ctracks
 {
-	z-index: 4;
+	z-index: 9;
```

Comment:
You don't have to increase the z-index to 9. It is just used for ordering (ETCS is more important).

Reply:
Good catch. ATB can share a value with any of the German national systems, as they will not have to override each other.

Reply:
I was aware of that @Nakaner and I did it in order to reserve some room. Currently I am in contact with a Belgian colleague and we are developing a tagging scheme for the Belgian and French systems. In this way further ensure to keep ETCS as the more important system, as the render scheme develops.
As soon as @DerDakon 's change request is re-approved, I hope to merge the pull request into the master.